create-customer/src/main/java/no/unit/nva/customer/create/CreateCustomerHandler.java

```diff
@@ -38,19 +55,25 @@ public CreateCustomerHandler() {
      * @param customerService customerService
      * @param environment   environment
      */
-    public CreateCustomerHandler(CustomerService customerService, Environment environment) {
-        super(Customer.class, environment, logger);
+    public CreateCustomerHandler(
+        CustomerService customerService,
+        CustomerMapper customerMapper,
+        Environment environment) {
+        super(CustomerDto.class, environment, logger);
         this.customerService = customerService;
+        this.customerMapper = customerMapper;
     }
 
     @Override
-    protected Customer processInput(Customer input, RequestInfo requestInfo, Context context)
+    protected CustomerDto processInput(CustomerDto input, RequestInfo requestInfo, Context context)
             throws ApiGatewayException {
-        return customerService.createCustomer(input);
+        CustomerDb customerDb = customerMapper.toCustomerDb(input);
+        CustomerDb createdCustomerDb = customerService.createCustomer(customerDb);
+        return customerMapper.toCustomerDto(createdCustomerDb);
```

Comment:
I love the following 3 code lines here, we can later put this in a  `Customer` business object as well.